README.md

```diff
@@ -1,2 +1,105 @@
-# procedure-module-pattern
-In a system that processes Messages, place each Procedure in a separate module.
+# Procedure Module pattern
+
+**"A module containing all Phases of a message processing Procedure."**
+
+The **Procedure Module** is the short name for the **Message Processing Procedure Module (MPPM)**.
+
+The **Procedure Module** is a structural code organization pattern.
+
+## Definitions
+
+**Message**: information sent to notifies that something has occurred, or to request for something to occur. Actions, Commands and Events are Messages.  
```

Comment:
I feel that, if a term is capitalized, it should have a definition.

Reply:
Rewrote this way:
In some contexts, Messages may also be called actions, commands or events.